addons/web_editor/models/ir_qweb_fields.py

```diff
@@ -20,11 +20,9 @@
 import requests
 from datetime import datetime
 from lxml import etree, html
-from PIL import Image as I
+from PIL import Image as Img  # 'Image' is already a class name in this file
```

Comment:
@odoo/rd-security This one is triggering the `ci/security` (for the 2 lines usage below).
Nothing dangerous, just a renaming.